quesma/ingest/insert_test.go

```diff
@@ -356,3 +358,81 @@ func dateTime(name string) *clickhouse.Column {
 		Modifiers: "CODEC(DoubleDelta, LZ4)",
 	}
 }
+
+func TestCreateTableIfSomeFieldsExistsInSchemaAlready(t *testing.T) {
```

Comment:
```suggestion
// TestCreateTableIfSomeFieldsExistsInSchemaAlready verifies the result `CREATE TABLE` statement when the ingested JSON doesn't contain *all* the columns from the table. In this case we should take the column data from schema and make sure the final statement includes all the columns. 
func TestCreateTableIfSomeFieldsExistsInSchemaAlready(t *testing.T) {
```

Reply:
this is quite complex case, I think this comment will explain it better than any name :)